stable/redis-ha/templates/redis-service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: {{ template "name" . }}-svc
```

Comment:
No need to have "svc" in the name for the service, since it's clear that it's 1) a service resource type and 2) the full path will be `{{ template "name" . }}.{{ .Release.Namespace }}.svc.cluster.local`, e.g. something like `unhinged-woodpecker-redis-ha.default.svc.cluster.local`.